src/main/java/frc/robot/RobotContainer.java

```diff
@@ -1,58 +1,377 @@
+
+    private final JoystickButton dampen = new JoystickButton(driver, XboxController.Button.kRightBumper.value);
+
+    private final POVButton up = new POVButton(driver, 90);
+    private final POVButton down = new POVButton(driver, 270);
+    private final POVButton right = new POVButton(driver, 180);
+    private final POVButton left = new POVButton(driver, 0);
+
+    /* Subsystems */
+    private final Swerve s_Swerve = new Swerve();
+    private final PoseEstimator s_PoseEstimator = new PoseEstimator();
+    private static Shooter shooter;
+
+
+  // Subsystems
+
+  // Controller
+  private static final CommandXboxController driverController =
+      new CommandXboxController(Constants.kDriverControllerPort);
+  private static final CommandXboxController operatorController =
+      new CommandXboxController(Constants.kOperatorControllerPort);
+
+  private static Intake intake;
+private static Arm arm = new Arm();
+
+
+  // Dashboard inputs
+
+  
+
+    /** The container for the robot. Contains subsystems, OI devices, and commands. */
+    public RobotContainer() {
+        s_Swerve.setDefaultCommand(
+            new TeleopSwerve(
+                s_Swerve, 
+                () -> -driver.getRawAxis(translationAxis), 
+                () -> -driver.getRawAxis(strafeAxis), 
+                () -> -driver.getRawAxis(rotationAxis), 
+                () -> centric,
+                () -> dampen.getAsBoolean(),
+                () -> 1 //speed multiplier 
+            )
+        );
 
-  /** The container for the robot. Contains subsystems, OI devices, and commands. */
-  public RobotContainer() {
-    // Configure the trigger bindings
-    configureBindings();
+        intake = new Intake();
+        shooter = new Shooter();
+
+
+
+
+
+        // Configure the button bindings
+        configureButtonBindings();
+    }
+
+    /**
+     * Use this method to define your button->command mappings. Buttons can be created by
+     * instantiating a {@link GenericHID} or one of its subclasses ({@link
+     * edu.wpi.first.wpilibj.Joystick} or {@link XboxController}), and then passing it to a {@link
+     * edu.wpi.first.wpilibj2.command.button.JoystickButton}.
+     */
+    
+  // private static final LEDs leds = new LEDs();
+  // public static final Arm arm = new Arm(operator.getHID(), leds);
+
+  // private static Intake intake;
+  // private static Autonomous autonomous;
+
+  // Dashboard inputs
+  // private final LoggedDashboardChooser<Command> autoChooser;
+
+  public Command ampButtonBinding() {
+    return new SequentialCommandGroup(
+        new ParallelCommandGroup(new InstantCommand(() -> shooter.setAmpSpeed())),
+        new InstantCommand(() -> arm.goToAmpPos()),
+        new WaitCommand(1.0),
+        new InstantCommand(() -> intake.intake()));
+  }
+
+  public Command shootButtonBinding() {
+    return new SequentialCommandGroup(
+        new ParallelCommandGroup(new InstantCommand(() -> shooter.setSpeakerSpeed())),
```

Comment:
Same question, why Instant in Parallel?

Reply:
Same response, we were working quickly and looked over removing the ParallelCommandGroup.